interchaintest/poa_group_test.go

```diff
@@ -115,7 +113,7 @@ func TestGroupPOA(t *testing.T) {
 
 	ctx, ic, client, _ := interchaintest.BuildInitialChain(t, chains, false)
 
-	user1Wallet, err := interchaintest.GetAndFundTestUserWithMnemonic(ctx, user1, accMnemonic, DefaultGenesisAmt, chain)
+	_, err = interchaintest.GetAndFundTestUserWithMnemonic(ctx, user1, accMnemonic, DefaultGenesisAmt, chain)
```

Comment:
if not using user1Wallet i think this line can just be removed

Reply:
We need to fund the account otherwise the proposal creation complains.